### PR TITLE
Creating new config docs

### PR DESCRIPTION
**Pull request by Kelvinrr**
Complete with a pretty new theming. 

Screen Shot 2021-04-07 at 5 29 10 PM

### CODE REVIEW

**Comment by Kelvinrr** [comment]
@AustinSanders For some reason I can't tag you as a reviewer but it would be helpful for both of ya'll to look through this.

File: docs/reference/tutorials.rst [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+        File  = ("mro", "kernels/ik/mro_hirise_v??.ti")
+      EndGroup
+    
+      Group = Selection
+        Match = ("Instrument","InstrumentId","CRISM")
+        File  = ("mro", "kernels/ik/mro_crism_v??.ti")
+      EndGroup
+    EndObject
+
+Here, like with most IK db files, the IK selection is instrument specific, so the ``Match`` key is used by ISIS to specify image label keywords that must match in order to select this kernel. These usually habe no ``makledb`` and are hand written as scpice is not needed to extract kernel times.
```
Comment (AustinSanders) [inline docs/reference/tutorials.rst:144]: habe -> have

**Review by AustinSanders — APPROVED** [review]
Overall looks good. Mostly just small typos, but since these are tutorials I don't know if those are even important fixes.

**Comment by Kelvinrr** [comment]
typos are pretty important fixes in docs I would say.

**Comment by Kelvinrr** [comment]
@AustinSanders Feel free to merge whenever (mostly to make sure if you permission to merge a PR)

**Pull request merged**